worldengine/world.py

```diff
@@ -416,22 +415,10 @@ def start_mountain_th(self):
         return self.elevation['thresholds'][2][1]
 
     def max_elevation(self):
```

Comment:
replace all calls to max_elevation to the .max(), saves us time calling a function just to get to numpy's builtin.